docs/development_suite/deploy/pipeline-based/index.md

```diff
@@ -0,0 +1,17 @@
+---
+id: pipeline-based
+title: Pipeline-based deployment strategy
+sidebar_label: Pipeline-based deployment strategy
+---
+
+**Pipeline-based deployment** is an approach in software deployment where updates and changes to applications or configurations are initiated and directly pushed from a CI/CD pipeline to the target environment. This method allows for immediate changes to be pushed directly to the target environment. It provides **real-time control and efficiency** for urgent patches and releases. This kind of strategy is often referred to as **push-based deployment**. 
+
+
+The currently supported tools for pipeline-based deployments in the Console are: 
+
+- **GitHub**
+- **Gitlab**
+- **Jenkins**
```

Comment:
link to thespecific configuration pages?

Reply:
we do not have all of them. This was one of the general TODO's (we only have jenkins and github actions)